Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM    golang:1.12.9
+FROM    golang:1.13
```

Comment:
In addition, we should explicitly specify the tag to use instead of using the shared tag.